pkg/pdutil/pd.go

```diff
@@ -61,15 +68,14 @@ var (
 		"shuffle-region-scheduler":     {},
 		"shuffle-hot-region-scheduler": {},
 	}
-	// TODO remove this, see https://github.com/pingcap/br/pull/555#discussion_r509855972
-	pdRegionMergeCfg = []string{
-		"max-merge-region-keys",
-		"max-merge-region-size",
-	}
-	pdScheduleLimitCfg = []string{
-		"leader-schedule-limit",
-		"region-schedule-limit",
-		"max-snapshot-count",
+	expectPDCfg = map[string]interface{}{
+		"max-merge-region-keys": 0,
+		"max-merge-region-size": 0,
+		// TODO remove this schedule-limits, see https://github.com/pingcap/br/pull/555#discussion_r509855972
```

Comment:
as the rationale keeping the two `-limit`s in #555 was

> I think we could leave a TODO here and try to merge this firstly... I'm afraid that if any error occurs during editing, we cannot merge this in the current unstable CI environment...

do we still need to keep them here?

Reply:
There are some misunderstanding. 
1. `leader-schedule-limit` and `region-schedule-limit` does need in previous and current version to reduce the schedule. I'll update the comment later.
2. Before https://github.com/tikv/pd/pull/3131/ `leader-schedule-limit` and `region-schedule-limit`  doesn't support ttl second.  I think we should wait PD merge 3131 then test. 

I want to make sure this PR is compatible with the old BR version and easy to support set pd ttl config. I'll file another PR to update config when 3131 merged.